### PR TITLE
Finish French Translation

### PR DESCRIPTION
**Pull request by JoeEnderman**
I may have messed with your slab and stair capitalization. If you want me to change it, I can. Just tell me how you want it.

### CODE REVIEW

File: locale/nether.fr.tr [hunk 1]
```diff
@@ -34,57 +34,57 @@ Unknown player position=Position du joueur inconnue
 
 A Deep Netherrack Wall=Un mur profond en Netherrack
 A Netherrack Wall=Un mur en Netherrack
-A finely finished block of solid Nether Basalt=Un bloc fini de basalte du Nether solide
-A rough cut solid block of Nether Basalt=Un bloc solide de basalte du Nether taillé à la hache
+A finely finished block of solid Nether Basalt.=Un bloc fini de basalte du Nether solide.
+A rough cut solid block of Nether Basalt.=Un bloc solide de basalte du Nether taillé à la hache.
```
Comment (Treer) [inline locale/nether.fr.tr:38]: The part on the left hand side of the '=' has to exactly match the string in the .lua file, so adding full-stops here (on the left of the '=') without also adding them to the .lua file will create mismatches and break it.

This is the same problem I was trying to explain with the lowercase 's', where I "lost you" :), but it's easily fixed.

**Comment by Treer** [comment]
I can't merge this Pull Request because it has conflicts and it's also trying to re-apply the changes from your last pull request (I'm not sure why git isn't automatically fixing this).

If you're comfortable with git you can resolve the conflicts. If not, I can cherry-pick those last 3 commits into a separate Pull request and merge that.

**Comment by JoeEnderman** [comment]
I tried for several days to learn how to use git and I never could get it to work, so I just manually move files, and it works. So, if you want to seperate the changes out?

**Comment by Treer** [comment]
Merged via  #68

**Pull request closed without merging**